cpp/modules/luma.gl/core/src/model.cpp

```diff
@@ -99,7 +112,31 @@ auto Model::_createBindGroupLayout(wgpu::Device device, const std::vector<Unifor
 }
 
 void Model::_setVertexBuffers(wgpu::RenderPassEncoder pass) {
-  for (int location = 0; location < this->_attributeTable->num_columns(); location++) {
-    pass.SetVertexBuffer(location, this->_attributeTable->column(location)->buffer());
+  int location = 0;
+  for (auto const& attribute : this->_attributeTable->columns()) {
+    pass.SetVertexBuffer(location, attribute->buffer());
+    location++;
+  }
+
+  if (this->_instancedAttributeTable != nullptr) {
+    for (auto const& attribute : this->_instancedAttributeTable->columns()) {
+      pass.SetVertexBuffer(location, attribute->buffer());
+      location++;
+    }
   }
 }
+
+auto Model::_getInputStepMode(const std::shared_ptr<garrow::Field>& field) -> wgpu::InputStepMode {
```

Comment:
All the fields (and only the fields) in the instanced table will have step mode `wgpu::InputStepMode::Instance`, so technically we don't need to store it in the metadata.